Equilobe.DailyReport.SL/JiraService.cs

```diff
@@ -87,6 +88,22 @@ public List<JiraUser> GetAllUsers(JiraRequestContext context)
             return GetClient(context).GetAllUsers();
         }
 
+        public List<JiraIssue> GetWorklogsForMultipleUsers(JiraRequestContext context, List<string> authors, DateTime fromDate, DateTime toDate)
+        {
+            var startDate = fromDate.ToString("yyyy-MM-dd");
+            var endDate = toDate.ToString("yyyy-MM-dd");
+            var worklogAuthors = string.Join(",", authors);
+
+            return GetClient(context).GetWorklogsForMultipleUsers(worklogAuthors, startDate, endDate);
+        }
+
+        public List<long> GetDeletedWorklogsIds(JiraRequestContext context, DateTime since)
+        {
+            var unixTimestamp = DateTimeHelpers.ToUnixTimestamp(since);
+
```

Comment:
Is this only possible with unix dateformat? The other api calls use the normal date.

Reply:
The doc is here 
https://developer.atlassian.com/cloud/jira/platform/rest/#api-group-Worklog

and it says, that the parameter since is:
The date and time, in UNIX timestamp format, after which deleted worklogs are returned.